SU-SWS/stanford_profile
# D8CORE-6217 Allow configuring maximum main menu depth

# READY FOR REVIEW

# Summary
- Allow site managers the ability to turn on the main menu drop down & choose how many levels to display

# Need Review By (Date)
- :shrug: 

# Urgency
- low

# Steps to Test
1. Checkout this branch
2. import configs & clear caches
3. Create several levels of menu items. (at least 4 levels)
4. turn on the drop down menu at `/admin/config/system/basic-site-settings`
5. observe the drop down menu shows all levels
6. set the "Maximum Menu Levels" so some value at `/admin/config/system/basic-site-settings`
7. observe the drop down shows only that many levels.
